extensions/big-data-cluster/src/main.ts

```diff
@@ -20,7 +20,6 @@ const kubectl = kubectlCreate(host, fs, shell, installDependencies);
 export let controller: MainController;
 
 export function activate(context: vscode.ExtensionContext) {
-    kubectl.checkPresent(CheckPresentMessageMode.Activation);
```

Comment:
Are you planning to hook this into the status page?

Reply:
we won't enable the popup experience, it will be displayed inline in the tool status table